### PR TITLE
ref: Clean up the release clutter

### PR DESCRIPTION
**Pull request by BYK**
Follow up to #1450.

### CODE REVIEW

File: .vscode/numbered-bookmarks.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (st0012) [inline .vscode/numbered-bookmarks.json:1]: @BYK can you remove this? thx

File: .github/workflows/prepare_raven_release.yml [hunk 2]
```diff
@@ -1,10 +1,7 @@
-name: Prepare Release
```
Comment (st0012) [inline .github/workflows/prepare_raven_release.yml:1]: I'd prefer keeping it for now for releasing new integrations in the future.

Comment (BYK) [reply .github/workflows/prepare_raven_release.yml:1]: We can add this back easily. I'd prefer if we nudge people to use the batch release. What do you think?

File: .vscode/numbered-bookmarks.json (hunk 1 above)
Comment (BYK) [reply .vscode/numbered-bookmarks.json:1]: Yup, sorry 😅

File: .github/workflows/prepare_raven_release.yml (hunk 2 above)
Comment (st0012) [reply .github/workflows/prepare_raven_release.yml:1]: ok let's drop it for now 👍

**Review by st0012 — APPROVED** [review]

**Comment by st0012** [comment]
@BYK Thanks!

**Pull request merged**